src/pages/messages/components/NavigationIconButton.tsx

```diff
@@ -0,0 +1,33 @@
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { IconButton, IconButtonProps } from '@mui/material';
+import { Link, LinkProps } from 'react-router-dom';
+
+interface NavigationIconButtonProps extends IconButtonProps, Pick<LinkProps, 'to'> {
+  navigateTo: 'previous' | 'next';
+}
+
+export const NavigationIconButton = ({ navigateTo, sx, ...rest }: NavigationIconButtonProps) => (
+  <IconButton
+    component={Link}
+    size="small"
+    sx={{
+      display: { xs: 'none', md: 'flex' },
+      alignSelf: 'center',
+      justifySelf: navigateTo === 'previous' ? 'start' : 'end',
```

Comment:
Dette er veldig tett knyttet til hvordan det er satt opp for nvi, så vi må huske å sjekke alle plasser dette brukes når vi gjør endringer her (som jeg mistenker vi kommer til å gjøre mye av når dette skal inn andre plasser 😅  )

Reply:
Ja, tenkte på det selv også. Men er vanskelig å gjøre det på en annen måte når man ikke vet hvordan det kommer til å se ut andre plasser😬

Reply:
jepp, fint å ha det sånn for nå tenker jeg